bundler.go

```diff
@@ -449,6 +450,10 @@ func (b *Bundler) bundle(e ConfigurationEnvironment) (err error) {
 		"TAGS=" + os.Getenv("TAGS"),
 	}
 
+	for k, v := range e.EnvironmentVariables {
```

Comment:
Also you'll have to check that `e.EnvironmentVariables != nil` before ranging into it otherwise we may get a panic here